README.md

```diff
@@ -25,9 +25,11 @@ gem 'twingly-search'
 require 'twingly/search'
 
 client = Twingly::Search::Client.new
```

Comment:
How about we show how to set the user agent here? Something like:

```Ruby
client = Twingly::Search::Client.new do |client|
  client.user_agent = "MyCompany/1.0"
end
```

I'm thinking we need to promote it if we want our customers to set a custom user agent.